### ISSUE TITLE
“Reply to” box would be useful for Photo and Post entity types

### ISSUE THREAD

**Issue by cdn** [idno/Known#2220]
## While trying to do this:

Reply to a tweet with a Photo

## I encountered this error:

No box to place the tweet URL

## Some other notes:

May also be useful in other circumstances

Give us some context: 

* It'd also be really handy if you could tell us the contents of your ```version.known``` file
0.9.9-a 2018041302 / 0.9.9 2018032701
* What database are you using? (e.g. mongo, mysql, postgres)
postgres, sqlite
* Any warnings or errors in your admin/diagnostics page?
* Bonus points - are you able to illustrate the issue with a unit test? If so, submit it as a pull request!

**Comment by cleverdevil** [comment]
I tend to agree: `in-reply-to` support should be available persistently across all post types, eventually.

**Comment by jgmac1106** [comment]
with a photo post you can not manually add the property...well you can..but only the photo title gets syndicated to Twitter so the reply isn't picked up.

**Comment by cdn** [comment]
Have made changes to [Photo](https://github.com/cdn/photo), and [Twitter](https://github.com/cdn/twitter) to get replies on Photo posts
